### PR TITLE
Exams by Matiushenko Kostia

### PR DESCRIPTION
**Pull request by jojonoskill**

### CODE REVIEW

File: Tasks/duplicate.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-  else {
-    res = [];
-    for (let i = 0; i < N; i++) {
-      res[i] = value;
-    }
-    return res;
+const duplicate = (value, N) => {
+  const result = [];
+  for (let i = 0; i < N; i++) {
+    result[i] = value;
```
Comment (tshemsedinov) [inline Tasks/duplicate.js:7]: Better use `push` mathod

File: Tasks/duplicate.js [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-duplicate = (value, N) => {
-  if (N <= 0) return [];
-  else {
-    res = [];
-    for (let i = 0; i < N; i++) {
-      res[i] = value;
-    }
-    return res;
+const duplicate = (value, N) => {
```
Comment (tshemsedinov) [inline Tasks/duplicate.js:4]: Use `camelCase` so `n` or `count`
```suggestion
const duplicate = (value, n) => {
```

File: Tasks/random.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
-    return min + Math.floor(Math.random() * (max - min + 1));
+const random = (min, max) => {
+  const correctionInt = 1;
+  if (!max) {
+    const randomFloat = Math.random() * (min + correctionInt);
+    return Math.floor(randomFloat);
   }
+  const randomFloat = min + Math.random() * (max - min + correctionInt);
+  return Math.floor(randomFloat);
 };
```
Comment (tshemsedinov) [inline Tasks/random.js:12]:
```js
const random = (min, max) => {
  const rnd = Math.random();
  return min + Math.floor(rnd * max);
};
```

Comment (jojonoskill) [reply Tasks/random.js:12]: https://github.com/jojonoskill/Exams/tree/exams

**Pull request closed without merging**